src/sentry/migrations/0423_remove_group_forecast_model.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.28 on 2023-04-18 21:12
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0422_remove_group_forecast_fk_constraint"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name="GroupForecast",
+        ),
+    ]
```

Comment:
This needs to be a state only operation for now. We just want to remove code references here, not the actual database table itself (that's step 3).

Even though this table is unused/empty, the problem here is that if you drop the table, then there's still code in production that thinks it exists. So if someone deletes a `Group`, then as part of that deletion django will attempt to query this table to see if there are any related rows that should be removed. Since the table doesn't exist, then we'll throw an exception and the deletion will fail.

Reply:
Thank you, I didn't notice it wasn't generated as state only